README.md

```diff
@@ -1,36 +1,148 @@
+<img src="http://i.imgur.com/LKv5XMA.jpg" />
+
 Boilerplate API
 ===
-A hapi boilerplate that makes creating APIs easy
+
+A general purpose boilerplate for building APIs with hapi
 
 ## Getting Started
-- Clone this repo: `git clone git@github.com:devinivy/boilerplate-api.git .`
-- Create a new repo on github, `new-api`.
-- Change origin to point to your new API: `git remote set-url origin git@github.com:your-username/new-api.git`
-- Replace references to "boilerplate" (`grep -rIni --color boilerplate *`), add model, and get developing.
+In this example our project is called `my-project` :droplet:
+
+```bash
+$ curl -LOk https://github.com/devinivy/boilerplate-api/archive/master.zip
+$ unzip master.zip
+$ mv boilerplate-api-master/ my-project/
+$ cd my-project/
+```
+
+Now install the dependencies and start running the server :ocean:
+
+```bash
+$ npm install
+$ npm start
+```
+
+If everything goes well you should see this :surfer:
+
+```bash
+> boilerplate-api@0.3.0 start /Users/maxfelker/boilerplate-api
+> node server
```

Comment:
`npm start`?  Just don't want people writing `node server.js` by mistake and it failing.